test-schema.json

```diff
@@ -86,6 +66,27 @@
                 }
             },
             "additionalProperties": false
+        },
+
+        "outputItem": {
```

Comment:
None of the tests actually prescribe output.  Given we don't yet know how output is going to be tested, maybe we just remove this (and `output`) for now.